lua/nvim-treesitter.lua

```diff
@@ -1,10 +1,16 @@
+--TODO(theHamsta): remove once stabilized!
+if not pcall(require,"vim.treesitter.query") then
```

Comment:
I am sure we can have a better check than just that.

Reply:
Maybe using the output of `:version`, or `has()` ?

Reply:
Can you suggest an improvement? With `has` we could only check for vim-patches and version can vary a lot depending on the built enviroment. Also: how can we compare Git SHAs?

This tests for that specific commit which I suppose is the oldest working version, but I can't tell whether we rely on even newer features.